include/hw_bridge.h

```diff
@@ -108,18 +108,16 @@ enum canID_t {
 
 namespace ARM {
 namespace ACTUATOR {
-enum actuatorID_t { TURNTABLE, SHOULDER, ELBOW, WRISTLEFT, WRISTRIGHT, CLAW };
-std::string stringifyActuatorID(actuatorID_t actuator);
+enum ActuatorId { TURNTABLE, SHOULDER, ELBOW, WRISTLEFT, WRISTRIGHT, CLAW };
 }  // namespace ACTUATOR
 namespace PID {
-typedef struct __attribute__((__packed__)) {
+using tuningApiPayload = struct __attribute__((__packed__)) TuningApiPayload {
   float value;
   bool velocity;
-  uint8_t actuatorID;
-} payload;
-enum parameter_t { P, I, D, DEADZONE, BIAS };
-std::string stringifyParam(parameter_t param);
-std::string stringifyVelPos(bool vel_pos);
+  HWBRIDGE::ARM::ACTUATOR::ActuatorId actuatorID;
+};
+std::string str(const tuningApiPayload& payload);
```

Comment:
hmm useless to return a const string? im suprised: 
    return type 'const std::string' (aka 'const basic_string<char>') is 'const'-qualified at the top level, which may reduce code readability without improving const correctness [readability-const-return-type,-warnings-as-errors]

Reply:
the reason is that returning a const string will prevent its resources from being moved

Reply:
Another reason is that "why should you prevent modification" in the first place. If the dev wants a const string, they can `const std::string s = funciton_that_returns_string();`

Reply:
you can read more about const correctness here: https://isocpp.org/wiki/faq/const-correctness#overview-const

Reply:
also u gotta remember that `std::string` is dynamically allocated so making it const is not a good idea

Reply:
Is this referring to the correct line of code? or is it supposed to refer to `std::string HWBRIDGE::ARM::PID::str(const TuningApiPayload& payload) `

Reply:
i agree about how returning a const string doesnt make sense. what do you mean by your last comment?

Reply:
which comment?

Reply:
Is this referring to the correct line of code? or is it supposed to refer to std::string HWBRIDGE::ARM::PID::str(const TuningApiPayload& payload)

Reply:
This is the prototype for that function?

Reply:
yeah

Reply:
so its referring to the same thing?

Reply:
see my recent comments on the implementation of this prototype.